### PR TITLE
Bug #206114 - WEB] Textbox Placeholders are not visible properly,,Bug #204466-[ [WEB-admin] IP Login > Learner > Enrollment Verification > Names and status are not readable, Bug #204907- WEB-ADMIN] IP Login> Prerak Orientation > Select Prerak > Status are not displaying properly ,Bug #206143 -  [ [WEB ADMIN] IP Login > Prerak And Learner List Name should be clickable and should work as view button

### PR DESCRIPTION
**Pull request by sonaliTekdi**
### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [ ] Code is formatted as per format decided
* [ ] Updated acceptance criteria in tracker
* [ ] Updated test cases in test-cases-tracker
Screenshot from 2023-10-10 14-54-41
Link -[Bug #206114](https://tracker.tekdi.net/issues/206114)
-----------------------------------------
 [WEB-admin] IP Login > Learner > Enrollment Verification > Names and status are not readable bug screenshot
Screenshot from 2023-10-10 17-34-11
Link - Bug [#204466](https://tracker.tekdi.net/issues/204466)
------------------------------------

[WEB-ADMIN] IP Login> Prerak Orientation > Select Prerak > Status are not displaying properly screenshot
Screenshot from 2023-10-10 18-39-53
Link- [Bug #204907](https://tracker.tekdi.net/issues/204907)

### CODE REVIEW

File: apps/front-end/src/pages/admin/beneficiaries/AdminBeneficiariesListTable.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ const columns = (t, navigate) => [
     },
   },
   {
-    name: t("PRERAK_ID"),
+    name: t("PRERAK_ID_1"),
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/admin/beneficiaries/AdminBeneficiariesListTable.js:85]: please confirm with dnyanesh sir

File: apps/front-end/public/locales/hi/translation.json [hunk 2]
```diff
@@ -744,7 +744,8 @@
   "PRERAK_GETTING_MARRIED": "शादी होना",
   "PRERAK_GUIDE_THEM_TOWARDS_FUTURE_GOALS": "उन्हें भविष्य के लक्ष्यों की ओर मार्गदर्शन करें",
   "PRERAK_HELP_GIRLS_ATTEND_EXAMS": "लड़कियों की परीक्षा में मदद करें",
-  "PRERAK_ID": "प्रेरक की ID",
+  "PRERAK_ID": "ID",
```
Comment (sagarkoshti1990) [inline apps/front-end/public/locales/hi/translation.json:747]: please confirm with dnyanesh sir

File: apps/front-end/public/locales/hi/translation.json [hunk 3]
```diff
@@ -575,12 +575,12 @@
...
   "LEARNERS": "शिक्षार्थी",
-  "LEARNERS_AGE": "शिक्षार्थी की आयु",
-  "LEARNERS_ID": "शिक्षार्थी की ID",
+  "LEARNERS_AGE": "आयु",
+  "LEARNERS_ID": "ID",
   "LEARNERS_IN_CAMP": "शिविर में शिक्षार्थी",
   "LEARNERS_INFO": "शिक्षार्थी की जानकारी",
   "LEARNERS_LIST": "शिक्षार्थी की सूची",
-  "LEARNERS_NAME": "शिक्षार्थी का नाम",
+  "LEARNERS_NAME": "नाम",
```
Comment (sagarkoshti1990) [inline apps/front-end/public/locales/hi/translation.json:583]: please confirm with dnyanesh sir

File: apps/front-end/public/locales/hi/translation.json [hunk 4]
```diff
@@ -575,12 +575,12 @@
   "LEARNER_DISTRIBUTION": "शिक्षार्थी वितरण",
   "LEARNER_NOT_AVAILABLE": "शिक्षार्थी उपलब्ध नहीं हैं।",
   "LEARNERS": "शिक्षार्थी",
-  "LEARNERS_AGE": "शिक्षार्थी की आयु",
-  "LEARNERS_ID": "शिक्षार्थी की ID",
+  "LEARNERS_AGE": "आयु",
+  "LEARNERS_ID": "ID",
```
Comment (sagarkoshti1990) [inline apps/front-end/public/locales/hi/translation.json:579]: please confirm with dnyanesh sir

File: apps/front-end/src/pages/admin/beneficiaries/enrollment/EnrollmentVerificationList.js [hunk 5]
```diff
@@ -89,10 +91,12 @@ const columns = (t, navigate, filter) => [
         return "-";
       }
     },
+    width: "150px",
   },
   {
-    name: t("PRERAK_ID"),
+    name: t("PRERAK_ID_1"),
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/admin/beneficiaries/enrollment/EnrollmentVerificationList.js:97]: please confirm with dnyanesh sir

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: apps/front-end/src/pages/admin/beneficiaries/AdminBeneficiariesListTable.js (hunk 1 above)
Comment (sonaliTekdi) [reply apps/front-end/src/pages/admin/beneficiaries/AdminBeneficiariesListTable.js]: changed PRERAK_ID_1 with PRERAK_ID

File: apps/front-end/public/locales/hi/translation.json (hunk 2 above)
Comment (sonaliTekdi) [reply apps/front-end/public/locales/hi/translation.json]: Rechange "PRERAK_ID": "प्रेरक की ID",

File: apps/front-end/public/locales/hi/translation.json (hunk 4 above)
Comment (sonaliTekdi) [reply apps/front-end/public/locales/hi/translation.json]: rechange  "LEARNERS_AGE" and
  "LEARNERS_ID" as per original text

File: apps/front-end/src/pages/admin/beneficiaries/enrollment/EnrollmentVerificationList.js (hunk 5 above)
Comment (sonaliTekdi) [reply apps/front-end/src/pages/admin/beneficiaries/enrollment/EnrollmentVerificationList.js]: leave name: PRERAK_ID,removed PRERAK_ID_1

File: apps/front-end/public/locales/hi/translation.json (hunk 3 above)
Comment (sonaliTekdi) [reply apps/front-end/public/locales/hi/translation.json]: rechange "LEARNERS_NAME": "शिक्षार्थी का नाम"

File: lib/common-lib/src/components/inputs/FloatingInput.js [hunk 6]
```diff
@@ -89,6 +89,10 @@ export default function FloatingInput({
         </FormControl.Label>
       )}
       <Input
+        _focus={{
+          bg: 'transparent'
+        }}
+        focusOutlineColor={'#9E9E9E'}
```
Comment (sagarkoshti1990) [inline lib/common-lib/src/components/inputs/FloatingInput.js:95]: use theme color

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: lib/common-lib/src/components/inputs/FloatingInput.js (hunk 6 above)
Comment (sonaliTekdi) [reply lib/common-lib/src/components/inputs/FloatingInput.js]: done

**Review by sagarkoshti1990 — APPROVED** [review]

**Pull request merged**